pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -423,6 +424,7 @@ func buildKubeApiserverConfig(
 	genericConfig.Authorizer = kubeAuthorizer         // this is used to fulfill the kube SAR endpoints
 	genericConfig.AdmissionControl = admissionControl
 	genericConfig.RequestContextMapper = requestContextMapper
+	genericConfig.DisabledPostStartHooks.Insert(rbacrest.PostStartHookName)
```

Comment:
This comes from the rbac registry validation. I.e. the upstream authorizer is not used. The validation complains in `ConfirmNoEscalation`. We can
- tweak the storage test to write more innocent values (if that is possible)
- ignore them completely in the storage test (and loosing test coverage)
- leave the bootstrap hook enabled.

@liggitt @enj what do you prefer? Patches for the rebase branch welcome.